<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="detectLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>detect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCameraBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close App</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognitionBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Face recognition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFaceBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Face</string>
    </property>
   </widget>
   <widget class="QLabel" name="peopleLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>number of people:  0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="facesLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>number of faces:  0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>name:  whose name ?? </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
